lineoperations/README

```diff
@@ -0,0 +1,226 @@
+===============
+Line Operations
+===============
+.. contents::
+
+About
+=====
+
+Line Operations is an assortment of simple line functions that can be applied to an open file.
+
+Features
+========
+
+* Remove Duplicate Lines, sorted
+* Remove Duplicate Lines, ordered
+* Remove Unique Lines
+* Remove Empty Lines
+* Remove Whitespace Lines
+* Sort Lines Ascending
+* Sort Lines Descending
+
+Usage
+=====
+
+After the plugins has been installed successfully, load the plugin via Geany's plugin manager and a new menu item in the Tools menu will appear. Click on each menu item to apply the operation on whole  file. See descriptions below to see operations for each menu item.
```

Comment:
I would wrap long lines at the 72th/80th column myself.  not sure if we require this though, @frlan